test/Test/Universum/Property.hs

```diff
@@ -0,0 +1,107 @@
+
+module Test.Universum.Property
+        ( hedgehogTestTree
+        ) where
+
+import Universum
```

Comment:
According to our style guide you should add empty line after `Universum` import.

Reply:
reviewed style guide, I think my code is now compliant